ui/raidboss/data/04-sb/eureka/eureka_anemos.ts

```diff
@@ -179,6 +179,9 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: { name: 'Pazuzu', capture: false },
       run: (data) => data.wraithCount = 0,
     },
+    // https://xivapi.com/LogMessage/916
+    // en: 7 minutes have elapsed since your last activity. [...]
+    // But no 0x29 log line; so have to use GameLog for now.
```

Comment:
There's just flat out no network packet for this, so I think the "for now" is a bit misleading. The client tracks the idle time locally, separate from the server, and that's what keys this message to fire.

Reply:
> There's just flat out no network packet for this, so I think the "for now" is a bit misleading. The client tracks the idle time locally, separate from the server, and that's what keys this message to fire.

Agree & will remove the "for now", but that raises an interesting -- if sort of collateral -- issue.  These GameLog lines use `code` 0039, which [cactbot calls](https://github.com/OverlayPlugin/cactbot/blob/448ecf83b59d287f484054cfbdac71f4849e7abc/resources/netlog_defs.ts#L116) a `message`.

I assumed `code` was coming from the network packet, but if these lines are (or can be) generated by ACT solely from memory reads and not a packet, I think(?) that would mean either ACT or the game client has some reference/table it's using to generate these codes? The LogGuide doesn't have much to say, but quisquous does have a 4+ year old [list](https://gist.github.com/quisquous/1a6ea3cf102c65e3c375186f1173dff3) of what some of these codes may correspond to.

I suppose where I'm going with this is that I'm continuing to run across `GameLog` triggers that don't have a corresponding 0x29 (or other) line, and, it sounds like, may not have an associated network packet at all.  Wondering if it's worth exploring whether OP could do a similar read of chat log lines from memory and write at least some (perhaps based on `type`, and that don't implicate privacy issues) as an OP log line, to avoid the issues of using suppressible `00` lines for triggers/timelines?

Reply:
Interesting!  That seems to check out with some of the other patterns I've seen, e.g., `0039` is used for a fair number of log messages, like "Triple Triad matches not allowed here", but `0839` is used for things like "You change to gunbreaker."  (Although, incidentally, I think `0839` is also the code used for zone seal/unseal messages too.)

Definitely interested in unpacking `code` more, just out of personal curiosity, but in terms of this PR, yeah, I don't think we need a custom log line for idle time, especially since it's the walking definition of a non-combat trigger.  I think we can live with this staying as a `GameLog` trigger for the foreseeable future.